docs/docs/guide/project/manifests.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Manifests
+description: Learn about the manifest files that Tuist uses to define projects and workspaces and configure the generation process.
+---
+
+# Manifests
+
+Tuist defaults to Swift files as the primary way to define projects and workspaces and configure the generation process. These files are referred to as **manifest files** throughout the documentation. 
+
+The decision of using Swift was inspired by the [Swift Package Manager](https://www.swift.org/documentation/package-manager/), which also uses Swift files to define packages. Thanks to the usage of Swift, we can leverage the compiler to validate the correctness of the content and reuse code across different manifest files, and Xcode to provide a first-class editing experience thanks to the syntax highlighting, auto-completion, and validation.
+
+> [!NOTE] CACHING
+> Since manifest files are Swift files that need to be compiled, Tuist caches the compilation results to speed up the parsing process. Therefore, you'll notice that the first time you run Tuist, it might take a bit longer to generate the project. Subsequent runs will be faster.
+
+## Project.swift
+
+The [`Project.swift`](/reference/project-description/project) manifest declares an Xcode project. The project gets generated in the same directory where the manifest file is located with the name indicated in the `name` property.
+
+```swift
+// Project.swift
+let project = Project(
+    name: "App",
+    targets: [
+        // ....
+    ]
+)
+```
+
+
+> [!WARNING] ROOT VARIABLES
+> The only variable that should be at the root of the manifest is `let project = Project(...)`. If you need to reuse code across various parts of the manifest, you can use Swift functions.
+
+## Workspace.swift
+
+By default, Tuist generates an [Xcode Workspace](https://developer.apple.com/documentation/xcode/projects-and-workspaces) containing the project being generated and the projects of its dependencies. If for any reason you'd like to customize the workspace to add additional projects or include files and groups, you can do so by defining a [`Workspace.swift`](/reference/project-description/workspace) manifest.
+
+```swift
+// Workspace.swift
+import ProjectDescription
+
+let workspace = Workspace(
+    name: "App-Workspace",
+    projects: [
+        "./App", // Path to directory containing the Project.swift file
+    ]
+)
+```
+
+> [!NOTE]
+> Tuist will resolve the dependency graph and include the projects of the dependencies in the workspace. You don't need to include them manually.
```

Comment:
I think we should explain what's the point of defining them here then

Reply:
Good idea. I added an additional sentence clarifying that.